api/test/e2e/route_test.go

```diff
@@ -224,6 +224,48 @@ func TestRoute_Update_Routes_With_Hosts(t *testing.T) {
 	}
 }
 
+func TestRoute_Patch(t *testing.T) {
+	tests := []HttpTestCase{
+		{
+			caseDesc:     "route patch",
```

Comment:
we should always create the related resource first. 

eg: the `route` resource

Reply:
fixed